ledger-tool/src/bigtable.rs

```diff
@@ -1216,21 +1245,57 @@ pub fn bigtable_process_command(ledger_path: &Path, matches: &ArgMatches<'_>) {
     });
 }
 
-fn missing_blocks(reference: &[Slot], owned: &[Slot]) -> Vec<Slot> {
+#[derive(Debug, PartialEq)]
+struct MissingBlocksData {
+    missing_blocks: Vec<Slot>,
+    superfluous_blocks: Vec<Slot>,
+    num_reference_blocks: usize,
+    num_owned_blocks: usize,
+}
+
+fn missing_blocks(
+    last_block_checked: &Slot,
```

Comment:
nit: deref in the caller and just pass `Slot` since `Slot` is a primitive type (u64) ? Can ignore this is we opt to move the derivation of this value inside the function